### PR TITLE
Essential Javascript Patterns

### PR DESCRIPTION
**Pull request by prashant-andani**
Adding a book in the Javascript Section on 
Essential Javascript Patterns by addyosmani

### CODE REVIEW

**Comment by prashant-andani** [comment]
Adding a book in the Javascript Section on
Essential Javascript Patterns by addyosmani

File: README.md [hunk 1]
```diff
@@ -94,6 +94,7 @@ Because we think it might help others as well we open-sourced it. If you know so
 * [JavaScript reference - MDN (Mozilla Developer Network)](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
 * [Eloquent Javascript](http://eloquentjavascript.net/)
 * [Babel.js - ES6 today](https://babeljs.io/)
+* [Essential Javascript Patters] (https://addyosmani.com/resources/essentialjsdesignpatterns/book/)
```
Comment (domnikl) [inline README.md]: Please remove the space here between the braces.

Comment (me-shaon) [inline README.md]: There is a typo: **Patters** should be **Patterns**

Comment (prashant-andani) [inline README.md]: @me-shaon thanks for the check on the typo.. Fixed it now

**Comment by prashant-andani** [comment]
@domnikl is this k to get merged?

**Comment by domnikl** [comment]
Yes, merged it. Thank you!

**Pull request merged**